ms2deepscore/utils.py

```diff
@@ -37,3 +39,32 @@ def load_spectra_as_list(file_name) -> List[Spectrum]:
     if isinstance(spectra, Generator):
         return list(spectra)
     return spectra
+
+
+@numba.jit(nopython=True)
+def scaled_intensity_sum(mz_values, intensities, min_mz=0, max_mz=1000, scaling=2):
+    """Compute a scaled intensity sum for all peaks of a spectrum.
+
+    Each peak will be scaled by the power of `scaling`.
```

Comment:
Seems to do what it says. Did this improve performance for you? The rational behind it is not directly apparent to me (but that is fine as long as you know)

Reply:
@florian-huber

Reply:
Ah I misunderstood what the purpose was, but now get it again. this was the function to remove spectra that did not have enough high intensity peaks, for spectra with many inchikeys for instance. Maybe quickly add that to the docstring as well